### DISCUSSION TITLE
Firecrawl is looking for cracked engineers!

### DISCUSSION THREAD

**Discussion by calebpeffer** [firecrawl/firecrawl#960] [Announcements]
Hey Firecrawl!  

The response we've seen from developers over the last 7.5 months has been staggering—19k GitHub stars! Firecrawl has turned into so much more than we expected when we created the repo and landing page as a weekend hackathon project!  

We're taking off like a rocket ship, and our small (but mighty) team of 6 can't keep holding everything together with duct tape. We have some big things coming in the pipeline, and we need **exceptional engineers to help us build the interconnection tissue between AI and the web.** We hope **one of those exceptional engineers could be you.**  

### **We have these roles. You should apply if:**  

- You code all the time and can demonstrate it with your GitHub.  
- You're startup-minded, ambitious, and want to get your hands dirty to build something exceptional.  
- You're a low-ego, team player who prioritizes the mission.  
- You love building.  
- You believe AI is the most impactful and important technology of our lifetime.  

### **Do not apply if:**  

- You don't build/code for fun.  
- You don't believe in open source.  

### **We're looking for:**  

1. [Senior Frontend Engineer](https://www.ycombinator.com/companies/firecrawl/jobs/kgo2jXR-senior-frontend-engineer)  
2. [Founding Web Automation Engineer](https://www.ycombinator.com/companies/firecrawl/jobs/hZHD0j6-founding-web-automation-engineer)  
3. [Founding Fullstack Engineer](https://www.ycombinator.com/companies/firecrawl/jobs/jyqltto-founding-fullstack-engineer)  

### **Why you should apply:**  
- We're growing really fast (**revenue, usage, and stars!**)  
- You'll learn a ton.  
- We play a game of GeoGuessr every day!  

Best,  
Caleb + the Firecrawl Team

**Comment by nikasgnk** [comment]
Good morning from Greece. I find the Firecrawl feature very important. I wish you the best in your work. I shared your post on the forum <https://linux-user.gr/>, as I am not a professional programmer myself, but I deal with data and metadata due to my work(Public Contracts) and studies (Information science).

**Comment by diepdo1810** [comment]
Hello **Firecrawl Team**. I am Vietnamese. I shared about your firecrawl on facebook group, many people are interested. This confirms your crawl is the best. I am a developer with more than 3 years of experience working with PHP, javascript. But I am not the person you are looking for. I write this comment because I wish the team good luck. Thanks for reading.

**Comment by alysonvilela** [comment]
Seems like I fit on the requirements. Hope we can talk soon

**Comment by devinat1** [comment]
Hi firecrawl! I fit the requirements for the founding web automation engineer role. I have spent the past two years building web automations and working on LLM integrated web agents. I hope to talk with you about this role.
